### ISSUE TITLE
Cannot find module 'rimraf'

### ISSUE THREAD

**Issue by camhart** [fruffin/serverless-dotnet#1]
Once I installed and setup the plugin I started getting the following error:

   ```
  Error --------------------------------------------------

     Cannot find module 'rimraf'

     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.

  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues

     Please report this error. We think it might be a bug.

  Your Environment Information -----------------------------
     OS:                 win32
     Node Version:       6.9.2
     Serverless Version: 1.4.0

```

If I manually install rimraf it resolves it.

**Comment by rymoore** [comment]
Same problem here, any ideas? Running on mac

**Comment by camhart** [comment]
Just manually install it and it'll work:

Npm install rimraf

**Comment by fruffin** [comment]
Should be fixed in 1.0.7 which was just published.
Thanks for the bug report!
